spec/libraries/duplicator_spec.rb

```diff
@@ -2,6 +2,28 @@
 require 'rails_helper'
 
 RSpec.describe Duplicator, type: :model do
+  describe '#time_shift' do
+    let(:duplicator) { Duplicator.new([], time_shift: 1.year) }
+
+    it 'returns the full time shifted date for dates below the cap' do
```

Comment:
Usually we use context to suggest the different boundaries when testing, maybe we can nest these 2 inside context? 

If you prefer something neater within the code blocks, you can define `subject { Duplicator.new([], time_shift: 1.year).time_shift(original_date) }`, then define `original_date` and `expected_date` as variables.